### DISCUSSION TITLE
CodePush availability with Codemagic

### DISCUSSION THREAD

**Discussion by icarusdust** [codemagic-ci-cd/codemagic-docs#2865] [Announcements]
CodePush is now integrated with Codemagic and it allows you to use it for your React Native projects. [Here](https://docs.codemagic.io/integrations/codepush-integration/) is more about how to configure it.

**Comment by arunkvkumar** [comment]
Hi,
How can we add this code-push-standalone cli. we are not getting proper command. i am getting below error
You are not currently logged in. Run the 'code-push-standalone login' command to authenticate with the CodePush server. if it is there demo with react native please help me
